homeassistant/__main__.py

```diff
@@ -77,8 +77,72 @@ def get_arguments():
         '--open-ui',
         action='store_true',
         help='Open the webinterface in a browser')
-
-    return parser.parse_args()
+    parser.add_argument(
+        '-v', '--verbose',
+        action='store_true',
+        help="Enable verbose logging to file.")
+    parser.add_argument(
+        '--pid-file',
+        metavar='path_to_pid_file',
+        default=None,
+        help='Path to PID file useful for running as daemon')
+    if os.name != "nt":
+        parser.add_argument(
+            '--daemon',
+            action='store_true',
+            help='Run Home Assistant as daemon')
+
+    arguments = parser.parse_args()
+    if os.name == "nt":
+        arguments.daemon = False
```

Comment:
I think that on Windows, arguments.daemon will not exist because of the above if.